packages/mephisto-review-hook/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mephisto-review-hook",
-  "version": "1.0.3",
+  "version": "2.0.0",
```

Comment:
I think it's better to go with 1.1.0 here instead of 2.0.0

Reply:
I used 2.0.0 to indicate a breaking change. Will 1.1 have the same effect?

Reply:
hmm, ok let's keep 2.0.0